### PR TITLE
YC-1162: Add user name in all emails

### PR DESCRIPTION
**Pull request by rbovard**
Fix https://jira.liip.ch/browse/YC-1162

### CODE REVIEW

File: geocity/apps/accounts/templates/registration/password_reset_email.html [hunk 1]
```diff
@@ -1,5 +1,4 @@
-{% load i18n %}{% autoescape off %}
-{% translate "Bonjour," %}
+{% load i18n %}{% autoescape off %}{% translate "Bonjour" %},
```
Comment (monodo) [inline geocity/apps/accounts/templates/registration/password_reset_email.html:1]: Don't you want to add full name also ?

File: geocity/apps/submissions/templates/submissions/emails/submission_classified_for_services.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{% load i18n %}{% autoescape off %}{% blocktranslate %}Bonjour{% endblocktranslate %},
+{% load i18n %}{% autoescape off %}{% blocktranslate %}Bonjour {{ name }}{% endblocktranslate %},
```
Comment (monodo) [inline geocity/apps/submissions/templates/submissions/emails/submission_classified_for_services.txt:1]: Is the fullname in `name `?

File: geocity/apps/accounts/templates/registration/password_reset_email.html (hunk 1 above)
Comment (rbovard) [reply geocity/apps/accounts/templates/registration/password_reset_email.html:1]: I didn't found where we call this template, it seems to be in django and not geocity.

If you can point me out something, I can take a look.

File: geocity/apps/submissions/templates/submissions/emails/submission_classified_for_services.txt (hunk 2 above)
Comment (rbovard) [reply geocity/apps/submissions/templates/submissions/emails/submission_classified_for_services.txt:1]: Yes, see here: https://github.com/yverdon/geocity/blob/develop/geocity/apps/submissions/services.py#L193

File: geocity/apps/accounts/templates/registration/password_reset_email.html (hunk 1 above)
Comment (monodo) [reply geocity/apps/accounts/templates/registration/password_reset_email.html:1]: Maybe take a look at https://github.com/django/django/blob/main/django/contrib/admin/templates/registration/password_reset_email.html

Comment (monodo) [reply geocity/apps/accounts/templates/registration/password_reset_email.html:1]: Maybe (not tested) you can use `{{ user.get_username }}`

**Pull request merged**